build.js

```diff
@@ -547,7 +547,7 @@ function dataToHtml(skeleton, rawBrowsers, tests, compiler) {
     body.append(testRow);
     // If this row has a different category to the last, add a title <tr> before it.
     if (t.category && (!testNum || t.category !== tests[testNum-1].category)) {
-      testRow.before('<tr class="category"><td colspan="' + (Object.keys(browsers).length+2) + '">' + capitalise(t.category) + '</td></tr>');
+      testRow.before('<tr class="category"><td colspan="' + (Object.keys(browsers).length+3) + '">' + capitalise(t.category) + '</td></tr>');
```

Comment:
I believe the "+3" is due to the test name, current browser result and spacer columns.